ArduPlane/commands_logic.cpp

```diff
@@ -1184,7 +1184,7 @@ void Plane::nav_script_time_done(uint16_t id)
 // support for NAV_SCRIPTING mission command and aerobatics in other allowed modes
 bool Plane::set_target_throttle_rate_rpy(float throttle_pct, float roll_rate_dps, float pitch_rate_dps, float yaw_rate_dps)
 {
-    if (!nav_scripting_active()) {
+    if (!nav_scripting_active() && !nav_scripting.enabled) {
```

Comment:
i think we can instead just remove both checks, and always fill in the structure. We're better off checking the enable when the data is used instead of when it is set

Reply:
done